deploy/docs/additional_prometheus_configuration.md

```diff
@@ -59,46 +63,74 @@ There are many pre-built libraries that the community has built to expose these,
 
 ### Set up a service monitor so that Prometheus pulls the data
 
-Service Monitors is how we tell Prometheus what endpoints and sources to pull metrics from. To define a Service Monitor, create a yaml file on disk with information templated as follows:
+To expose metrics to prometheus, you need to have some service. Let's say here is our example service configuration:
 
+```yaml
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: example-metrics
+  # This is important, because prometheus matches service via labels
+  labels:
+    app: example-metrics
+spec:  # Should match your deployment ;)
```

Comment:
I'd loose the `;)` at the end, let's try to just be informative.

Reply:
sure, it took me a lot of time to figure this out :D

Reply:
It's just obvious :)